osu.Game/Skinning/SkinImporter.cs

```diff
@@ -20,17 +20,21 @@
 
 namespace osu.Game.Skinning
 {
-    public class SkinImporter : RealmArchiveModelManager<SkinInfo>
+    public class SkinImporter : RealmArchiveModelImporter<SkinInfo>
     {
         private const string skin_info_file = "skininfo.json";
 
         private readonly IStorageResourceProvider skinResources;
 
+        private readonly ModelManager<SkinInfo> modelManager;
+
         public SkinImporter(Storage storage, RealmAccess realm, IStorageResourceProvider skinResources)
             : base(storage, realm)
         {
             this.skinResources = skinResources;
 
+            modelManager = new ModelManager<SkinInfo>(storage, realm);
```

Comment:
A little perplexed by this, though. `SkinManager` (which is a `ModelManager<SkinInfo>`) creates this `SkinImporter`, which in turn creates its own `ModelManager` which is *not* the original `SkinManager`? Was this intentional?